### DISCUSSION TITLE
WindowBuilder removed and other churn - why?

### DISCUSSION THREAD

**Discussion by John-Nagle** [rust-windowing/winit#3951] [General]
WindowBuilder has been removed, Why was such a major breaking change made? This requires changes at three levels in five crates for no good reason.

**Comment by kchibisov** [comment]
It was renamed to `WindowAttributes`, I understand that you generally want to complain based on previous discussions, but the reason is that it's not really a builder anymore.

If you want something more stable, use more stable crates, which winit won't be until 1.0.
